app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -69,9 +70,11 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        // Handle action bar item clicks here. The action bar will
-        // automatically handle clicks on the Home/Up button, so long
-        // as you specify a parent activity in AndroidManifest.xml.
+        /**
+         *  Handle action bar item clicks here. The action bar will
+         *  automatically handle clicks on the Home/Up button, so long
+         *  as you specify a parent activity in AndroidManifest.xml.
+         */
```

Comment:
The structure of multiline comments should be such as per convention

https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
I recommend one more change... JavaDoc comments belong above a function, so move the /** */ comment above the function signature line, override fun onOptionsItemSelected.